infrastructure/metrics/src/test/java/tech/pegasys/teku/infrastructure/metrics/MetricsConfigTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Consensys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.metrics;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableSet;
+import java.util.EnumSet;
+import java.util.Set;
+import org.hyperledger.besu.metrics.StandardMetricCategory;
+import org.hyperledger.besu.plugin.services.metrics.MetricCategory;
+import org.junit.jupiter.api.Test;
+
+class MetricsConfigTest {
+
+  @Test
+  public void metricsConfigHasExpectedDefaultCategories() {
```

Comment:
Adding this here just in case someone touches `MetricsConfig.DEFAULT_METRICS_CATEGORIES`